components/shared/CustomButton.tsx

```diff
@@ -32,24 +38,21 @@ const CustomButton = ({
   outlineStyle,
 }: CustomButtonProps) => {
   return (
-    <TouchableOpacity
-      onPress={onPress}
-      disabled={disabled}
-      
-    >
-
+    <TouchableOpacity onPress={onPress} disabled={disabled}>
       <LinearGradient
-        start={{ x: 0.1, y: 0.7 }}
-        colors={['#0C8B02', '#45BB3B', '#10D101']}
+        colors={["#0C8B02", "#45BB3B", "#10D101"]}
+        // start={{ x: 0.1, y: 0.7 }}
```

Comment:
Is this piece of code going to be used hence commenting it?

Reply:
Yes, the code will still be used.